tests/SprykerEcoTest/Zed/Unzer/Business/UnzerFacade/CreateUnzerCredentialsTest.php

```diff
@@ -0,0 +1,297 @@
+ * @group Unzer
+ * @group Business
+ * @group UnzerFacade
+ */
+class CreateUnzerCredentialsTest extends UnzerFacadeBaseTest
+{
+    /**
+     * @return void
+     */
+    public function testCreateUnzerCredentialsReturnsSuccessfulUnzerCredentialsResponseWhileValidUnzerCredentialsGiven(): void
+    {
+        // Arrange
+        $this->tester->ensureUnzerCredentialsTableIsEmpty();
+        $unzerCredentialsTransfer = (new UnzerCredentialsBuilder([
+            UnzerCredentialsTransfer::ID_UNZER_CREDENTIALS => null
+        ]))
+            ->withUnzerKeypair([
+                UnzerKeypairTransfer::KEYPAIR_ID => null
+            ])->build();
+
+        // Act
+        $unzerCredentialsResponseTransfer = $this->tester
+            ->getFacade()
+            ->createUnzerCredentials($unzerCredentialsTransfer);
+
+        // Assert
+        $this->assertTrue($unzerCredentialsResponseTransfer->getIsSuccessful());
+        $this->assertSame($unzerCredentialsTransfer->getType(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getType());
+        $this->assertSame($unzerCredentialsTransfer->getParentIdUnzerCredentials(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getParentIdUnzerCredentials());
+        $this->assertNotNull($unzerCredentialsResponseTransfer->getUnzerCredentials()->getIdUnzerCredentials());
+        $this->assertSame($unzerCredentialsTransfer->getConfigName(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getConfigName());
+        $this->assertNotNull($unzerCredentialsResponseTransfer->getUnzerCredentials()->getUnzerKeypair()->getKeypairId());
+        $this->assertSame($unzerCredentialsTransfer->getUnzerKeypair()->getPublicKey(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getUnzerKeypair()->getPublicKey());
+        $this->assertSame($unzerCredentialsTransfer->getUnzerKeypair()->getPrivateKey(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getUnzerKeypair()->getPrivateKey());
+        $this->assertSame($unzerCredentialsTransfer->getMerchantReference(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getMerchantReference());
+        $this->assertSame($unzerCredentialsTransfer->getParticipantId(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getParticipantId());
+        $this->assertSame($unzerCredentialsTransfer->getKeypairId(), $unzerCredentialsResponseTransfer->getUnzerCredentials()->getKeypairId());
+    }
+
+    /**
+     * @return void
+     */
+    public function testCreateUnzerCredentialsThrowsNullValueExceptionWhileUnzerKeypairIsMissing(): void
+    {
+        // Arrange
+        $this->tester->ensureUnzerCredentialsTableIsEmpty();
+        $unzerCredentialsTransfer = (new UnzerCredentialsBuilder())->build();
+
+        // Assert
+        $this->expectException(NullValueException::class);
+        $this->expectExceptionMessage('Property "unzerKeypair" of transfer `Generated\Shared\Transfer\UnzerCredentialsTransfer` is null.');
```

Comment:
this check is not needed
previous is enough

Reply:
Removed.